clio-server/src/main/scala/org/broadinstitute/clio/server/webservice/IndexWebService.scala

```diff
@@ -10,6 +10,12 @@ import org.broadinstitute.clio.transfer.model.ApiConstants._
 import org.broadinstitute.clio.transfer.model._
 import org.broadinstitute.clio.util.model.UpsertId
 
+object IndexWebService {
+
+  def decodeSlashes(s: String): String =
+    s.replace("%2F", "/")
```

Comment:
I thought the algorithm would be more complex to support % quoting. Isn’t there a general method that does URL deciding.

Reply:
I was worried about cases like this: ```
Uri("%2520").path.head shouldEqual "%20"
Uri("/%2F%5C").path shouldEqual Path / """/\"""
```
since what we have is a string that has already been Uri decoded (excepting forward slashes) once. I figured just decoding the slash would leave us with fewer unsupported cases than double decoding